modules/misunderstood/src/views/full/index.tsx

```diff
@@ -0,0 +1,236 @@
+    await this.setEventIndex(0)
+  }
+
+  setEventIndex = async (selectedEventIndex: number) => {
+    const { events } = this.state
+    if (selectedEventIndex >= events.length) {
+      selectedEventIndex = events.length - 1
+    }
+    await this.setStateP({ selectedEventIndex, selectedEvent: null })
+    if (events && events.length) {
+      const event = await this.fetchEvent(events[selectedEventIndex].id)
+      await this.setStateP({ selectedEvent: event })
+    }
+  }
+
+  skipCurrentEvent = () => {
+    const { selectedEventIndex, eventCounts, selectedStatus } = this.state
+    if (
+      selectedStatus === FLAGGED_MESSAGE_STATUS.new &&
+      eventCounts &&
+      selectedEventIndex < eventCounts[selectedStatus]
+    ) {
+      return this.setEventIndex(selectedEventIndex + 1)
+    }
+  }
+
+  async alterNewEventsList(oldStatus: FLAGGED_MESSAGE_STATUS, newStatus: FLAGGED_MESSAGE_STATUS) {
+    // do some local state patching to prevent unneeded content flash
+    const { eventCounts, selectedEventIndex, events, selectedEvent } = this.state
+    const newEventCounts = {
+      ...eventCounts,
+      [oldStatus]: eventCounts[oldStatus] - 1,
+      [newStatus]: (eventCounts[newStatus] || 0) + 1
+    }
+    await this.setStateP({
+      eventCounts: newEventCounts,
+      selectedEvent: null,
+      events: events.filter(event => event.id !== selectedEvent.id)
+    })
+
+    // advance to the next event
+    await this.setEventIndex(selectedEventIndex)
+
+    await this.updateEventsCounts()
+  }
+
+  deleteCurrentEvent = async () => {
+    const { selectedEvent } = this.state
+
+    if (selectedEvent) {
+      await this.apiClient.updateStatus(selectedEvent.id, FLAGGED_MESSAGE_STATUS.deleted)
+
+      return this.alterNewEventsList(FLAGGED_MESSAGE_STATUS.new, FLAGGED_MESSAGE_STATUS.deleted)
+    }
+  }
+
+  undeleteEvent = async (id: string) => {
+    const { selectedEvent } = this.state
+
+    if (selectedEvent) {
+      await this.apiClient.updateStatus(id, FLAGGED_MESSAGE_STATUS.new)
+
+      return this.alterNewEventsList(FLAGGED_MESSAGE_STATUS.deleted, FLAGGED_MESSAGE_STATUS.new)
+    }
+  }
+
+  amendCurrentEvent = async (resolutionData: ResolutionData) => {
+    const { selectedEvent } = this.state
+
+    await this.apiClient.updateStatus(selectedEvent.id, FLAGGED_MESSAGE_STATUS.pending, resolutionData)
+    return this.alterNewEventsList(FLAGGED_MESSAGE_STATUS.new, FLAGGED_MESSAGE_STATUS.pending)
+  }
+
+  applyAllPending = async () => {
+    await this.apiClient.applyAllPending()
+    await this.updateEventsCounts()
+    return this.setEventsStatus(FLAGGED_MESSAGE_STATUS.applied)
+  }
+
+  render() {
+    const {
+      loaded,
+      languages,
+      language,
+      eventCounts,
+      selectedStatus,
+      events,
+      selectedEventIndex,
+      selectedEvent
+    } = this.state
+
+    const dataLoaded = selectedStatus === FLAGGED_MESSAGE_STATUS.new ? selectedEvent : events
+
+    return (
+      <Container>
```

Comment:
Container has the property `sidePanelWidth` that you can set to something bigger than the default (300px), since your components are overflowing a bit

Reply:
Thanks, was going to ask for how to handle it!